tests/test_ase.py

```diff
@@ -25,7 +25,7 @@ def get_numeric_force(atoms, eps):
 class TestASE(unittest.TestCase):
 
     def setUp(self):
-        self.model = torchani.models.ANI1x().double()[0]
```

Comment:
Is the user still allowed to do `torchani.models.ANI1x().double()[0]`, if they want, as before?

Reply:
Yes I think so, it should work same as before

Reply:
Looks like there is a slight difference between both ways of doing this for some reason, I'll look into it

Reply:
There is no difference, you can do both, but for some reason assertEqual() says thay are different, but it says they are different also for 2 instances of ani1x, no idea why it behaves like that.